web/admin/src/utils/drag.ts

```diff
@@ -200,3 +200,19 @@ export function getSiblingItemIds(
 
   return result;
 }
+
+export const collapseAllFolders = (
```

Comment:
在 drag.ts 文件中，添加了一个新的函数 `collapseAllFolders`，用于折叠或展开所有文件夹。这是一个有用的功能增强。

```suggestion
// 添加 collapseAllFolders 函数用于折叠或展开所有文件夹
export const collapseAllFolders = (
  list: TreeItems<ITreeItem>,
  collapsed: boolean,
): TreeItems<ITreeItem> => {
  return list.map(it => ({
    ...it,
    collapsed: it.type === 1 ? collapsed : it.collapsed,
    children: it.children
      ? (collapseAllFolders(
          it.children as TreeItems<ITreeItem>,
          collapsed,
        ) as ITreeItem[])
      : it.children,
  }));
};